src/core/CheckBox/CheckBox/__tests__/CheckBox.test.tsx

```diff
@@ -0,0 +1,15 @@
+import { render } from '@testing-library/react';
+import React from 'react';
+import { KuberaThemeProvider } from '../../../../theme';
+import { CheckBox } from '../CheckBox';
+
+describe('CheckBox', () => {
+  it('Renders', () => {
+    const { container } = render(
+      <KuberaThemeProvider platform="kubera-chaos">
+        <CheckBox checked={false} disabled={false} />
+      </KuberaThemeProvider>
+    );
+    container.querySelector('input');
```

Comment:
needs better test cases to check it's various states like disabled state or checked and unchecked states

Reply:
@arkajyotiMukherjee Can you please give more details of what you need here?

Reply:
This is not of need basis...This would be more dependent on how you have written the code @Hasmik8 
That will determine what testcases are needed